src/covariances.jl

```diff
@@ -0,0 +1,87 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    design_matrix(comps)
+    
+Converts a vector of Composition objects into a Matrix of numbers.
+"""
+function design_matrix(comps)
+    matrix = reduce(hcat, components.(comps)[:,1])'
+    return matrix
```

Comment:
```suggestion
    reduce(hcat, components.(comps)[:,1])'
```

Reply:
Are you sure we need the `[:,1]` there?

Reply:
I've tested without `[:, 1]` and with `[:, :]` and `[:]`. Using `[:, 1]` is the only thing that worked for here.

Reply:
You're right. I was using `[:, 1]` because I was passing the DataFrame (output of readcoda) instead a vector of compositions. Solving it locally and soonly I'll push it